odoo/addons/base/models/res_company.py

```diff
@@ -195,6 +197,13 @@ def _company_default_get(self, object=False, field=False):
         _logger.warning("The method '_company_default_get' on res.company is deprecated and shouldn't be used anymore")
         return self.env.company
 
+    @api.depends('company_details')
+    def _compute_empty_company_details(self):
+        #In recent change when an html field is empty a <p> balise remains with a <br> in it,
+        #but when company details is empty we want to put the info of the company
+        for record in self:
+            record.is_company_details_empty = record.company_details and not html2plaintext(record.company_details)
```

Comment:
```suggestion
            record.is_company_details_empty = not html2plaintext(record.company_details or '')
```
This should work, and also cover the case where `company_details` is actually empty.